src/main/java/codeu/model/data/MessageTimeComparator.java

```diff
@@ -0,0 +1,14 @@
+package codeu.model.data;
+
+import java.util.Comparator;
+
+//This comparator should be used to sort message objects by the time they were sent
+public class MessageTimeComparator implements Comparator<Message>
+{
+    // Used for sorting in descending order of time
+    @Override
+    public int compare(Message a, Message b)
+    {
+        return (b.getCreationTime().isAfter(a.getCreationTime())) ? -1 : 1;
```

Comment:
You can use this instead: https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#compareTo-java.time.Instant-

Moreover, you can actually just override the compareTo method in the Message class and then you wouldn't need to create an instance of this comparator. (Of course, with the caveat that if Messages need to be compared in some way in the future, it might conflict with that definition. I think it's fine for now)